mora/service/org.py

```diff
@@ -761,3 +765,77 @@ def get_org_unit_history(unitid):
                                unit_registrations))
 
     return flask.jsonify(history_entries)
+
+
+@blueprint.route('/o/<uuid:orgid>/address_autocomplete/')
+@util.restrictargs('global', required=['q'])
+def address_autocomplete(orgid):
+    """Perform address autocomplete
+    :param orgid: The UUID of the organisation
+
+    .. :quickref: Address; Autocomplete
+
+    :queryparam str q: A query string to be used for lookup
+    :queryparam uuid global: Whether or not the lookup should be in the entire
+        country, or contained to the municipality of the organisation
+
+    **Example Response**:
+
+    :<jsonarr uuid uuid: A UUID of a DAR address
+    :<jsonarr str name: A human readable name for the address
+
+    .. sourcecode:: json
+
+      [
+        {
+          "location": {
+            "uuid": "f0396d0f-ef2d-41e5-a420-b4507b26b6fa"
+            "name": "Rybergsvej 1, Sønderby, 5631 Ebberup"
+          }
+        },
+        {
+          "location": {
+            "uuid": "0a3f50cb-05eb-32b8-e044-0003ba298018"
+            "name": "Wild Westvej 1, 9310 Vodskov"
+          }
+        }
+      ]
+    """
+    q = flask.request.args['q']
+    global_lookup = flask.request.args.get('orgid')
```

Comment:
`s/orgid/global/`? I think we need a test for this 🙂 Also, we might want to do a bit more than merely check for presence and truth value. For example:

    is_global = flask.request.args.get('global', False, bool)

Not sure which works best, `bool` or `int`, but Werkzeug's docs has [this](http://werkzeug.pocoo.org/docs/0.14/datastructures/#werkzeug.datastructures.TypeConversionDict.get) and [this](http://werkzeug.pocoo.org/docs/0.14/datastructures/#werkzeug.datastructures.MultiDict.get) to offer.